src/server.rs

```diff
@@ -104,149 +98,149 @@ impl ServerPlugin {
     }
 
     /// Increments current server tick which causes the server to replicate this frame.
-    pub fn increment_tick(mut tick: ResMut<RepliconTick>) {
-        tick.increment();
-        trace!("incremented {tick:?}");
+    pub fn increment_tick(mut replicon_tick: ResMut<RepliconTick>) {
+        replicon_tick.increment();
+        trace!("incremented {replicon_tick:?}");
     }
 
     fn acks_receiving_system(
-        mut acked_ticks: ResMut<AckedTicks>,
-        mut ticks_map: ResMut<TicksMap>,
+        change_tick: SystemChangeTick,
         mut server: ResMut<RenetServer>,
-        mut entity_map: ResMut<ClientEntityMap>,
+        mut clients_info: ResMut<ClientsInfo>,
     ) {
-        for client_id in server.clients_id() {
-            while let Some(message) = server.receive_message(client_id, REPLICATION_CHANNEL_ID) {
-                match bincode::deserialize::<RepliconTick>(&message) {
-                    Ok(tick) => {
-                        let acked_tick = acked_ticks.0.entry(client_id).or_default();
-                        if *acked_tick < tick {
-                            *acked_tick = tick;
-                            entity_map.cleanup_acked(client_id, *acked_tick);
-                            trace!("client {client_id} acknowledged {tick:?}");
+        for client_info in clients_info.iter_mut() {
+            while let Some(message) =
+                server.receive_message(client_info.id, ReplicationChannel::Reliable)
+            {
+                match bincode::deserialize::<u16>(&message) {
+                    Ok(update_index) => {
+                        let Some((tick, entities)) =
+                            client_info.update_entities.remove(&update_index)
+                        else {
+                            error!(
```

Comment:
I agree here, bit I would prefer to address it later since it should be done in the whole codebase. Opened issue: https://github.com/lifescapegame/bevy_replicon/issues/127.